### ISSUE TITLE
Will this be upgraded to Elm 0.19?

### ISSUE THREAD

**Issue by eelcoh** [elm-community/lazy-list#12]

**Comment by mgold** [comment]
It was merged into `elm-test`, its main (only?) dependent.

The lazy package which did implicit memoization will not be updated for 0.19. That's okay for `elm-test` but probably a big regression for other users.

Do you have a use for this package outside of `elm-test`?

**Comment by mgold** [comment]
It was merged into `elm-test`, its main (only?) dependent.

The lazy package which did implicit memoization will not be updated for 0.19. That's okay for `elm-test` but probably a big regression for other users.

Do you have a use for this package outside of `elm-test`?

**Comment by eelcoh** [comment]
No, it was actually used in the test suite of https://github.com/wernerdegroot/listzipper . In otder to port that library, the lazy list is required. Will look at elm-test for this now. Thanks.
